Add awesome list about CRI-O

saschagrunert opened the pull request:
The idea of an *Awesome List* about CRI-O was born as a follow up of #2337 and should help others to find input about their use cases.

We may add further links to this PR.
What do you think about it?

vrothberg commented on awesome.md:
```diff
@@ -0,0 +1,31 @@
+
+This list follows the [awesome
+manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md), and
+collects every useful resource about CRI-O.
```
I am certain the list does not include *every* useful resource. Let's relax that a bit.

TomSweeneyRedHat replied:
s/every useful resource/useful resources/  Once we get to more than a dozen or so, we could add "many useful".

saschagrunert replied:
Sounds good, I change it as suggested by @TomSweeneyRedHat.

vrothberg commented on awesome.md:
```diff
@@ -0,0 +1,31 @@
+
+### Videos
+
+- [CRI-O: All the Runtime Kubernetes Needs](https://www.youtube.com/watch?v=NVa8xR4Eyj8)
```
A date would be nice here.

TomSweeneyRedHat replied:
and for the articles too, good point @vrothberg

saschagrunert replied:
Added the date as suggested.

vrothberg commented on awesome.md:
```diff
@@ -0,0 +1,31 @@
+
+### Community
+
+- [Kubernetes Slack #cri-o](https://slack.com/app_redirect?channel=CAZH62UR1)
```
There's also the cri-o channel on freenode. It's mostly spam from a bot but some folks reach out on freenode.

saschagrunert replied:
Added it but I cannot link it to somewhere. Do you have any suggestions?

vrothberg replied:
Maybe [https://webchat.freenode.net/](https://webchat.freenode.net/)?

TomSweeneyRedHat commented on awesome.md:
```diff
@@ -0,0 +1,31 @@
+
+### Websites
+
+- [cri-o.io (official website)](https://cri-o.io/)
```
I'm not sure if this list belongs here or on the cri-o.io website, but if it does end up here, we ought to add a link from the site to this page.  The advantage of keeping it here is it would more likely be updated by contributors.

saschagrunert commented:
> Currently, the list of links is very short which I'd like to have extended before merging. There are blogs from quite a bunch of contributors including Red Hat, Intel, SUSE, openSUSE Kubic and likely some of the individual contributors. The CNCF announcements would be nice to add as well.

I took more time to find some more resources and added them to the list. Feel free to provide further input. :+1: 

> [#2337 (comment)](https://github.com/cri-o/cri-o/issues/2337#issuecomment-491498508) seems to relate to tutorials; maybe we can add a dedicated section the file here to make it easier to brain-parse?

Yes sure, I added a tutorials section and moved the link to it.

umohnani8 commented:
LGTM

haircommander commented:
LGTM, waiting on a head nod from @TomSweeneyRedHat and then I'll flag it :)

haircommander commented:
/lgtm
